runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -66,7 +66,7 @@
      * A view of streams in the Corfu server instance.
      */
     @Getter(lazy = true)
-    private final MultiStreamView streamsView = new MultiStreamView(this);
+    private final MultiStreamView multiStreamsView = new MultiStreamView(this);
```

Comment:
Ok, then you're going to undo this change, right? If you're going to break applications, you might as well produce the intended change...